starlette/middleware/base.py

```diff
@@ -144,7 +144,7 @@ async def send_no_error(message: Message) -> None:
             async def coro() -> None:
                 nonlocal app_exc
 
-                async with send_stream:
+                async with send_stream, recv_stream:
```

Comment:
Since streams are unbuffered, it probably makes sense to close `recv` if we closed `send` 🤔

Reply:
Moved this change (as more invasive) to https://github.com/encode/starlette/pull/2778